src/main/java/com/github/algobot76/surabaya/SurabayaAnalyzer.java

```diff
@@ -0,0 +1,30 @@
+package com.github.algobot76.surabaya;
+
+import com.github.algobot76.surabaya.util.Project;
+import com.github.algobot76.surabaya.util.StaticAnalysisVisitor;
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.visitor.VoidVisitor;
+import com.github.javaparser.utils.CodeGenerationUtils;
+import com.github.javaparser.utils.SourceRoot;
+
+public class SurabayaAnalyzer {
```

Comment:
I don't think this file should be at the same directory level as `SurayabaApplication`. Maybe we should put it in `util` package. Also, the name might be a bit too long. How about renaming it as `Analyzer`?